define: &motorPins [0]

steer_center : 380
steer_limit : 100
steer_dir : -1
speed_center: 0
speed_limit: 4095

has_steer : 0
isDCSteer : 0

i2caddr0: 96
i2caddr1: 96

k_steer: 2.5
k_throttle: 0.2
